__tests__/integrations/Dialog/ConsultationDialog.test.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable dot-notation */
+const supertest = require("supertest");
+const faker = require("faker");
+const app = require("../../../src/app");
+
+const request = supertest(app);
+
+let authorization;
+let userId;
+let dialogId;
+
+const speech = faker.lorem.sentence();
+const answer = faker.lorem.sentence();
+beforeAll(async () => {
+  const response = await request.post("/v1/register").send({
+    name: faker.name.findName(),
+    email: faker.internet.email(),
+    password: faker.internet.password(),
+  });
+  authorization = `Bearer ${response.body.token}`;
+
+  userId = response.body.user["_id"];
+
+  const dialog = await request
+    .post("/v1/dialog")
+    .set("Authorization", authorization)
+    .send({ speech, answer });
+
+  await request
+    .post("/v1/dialog")
+    .set("Authorization", authorization)
+    .send({ speech: faker.lorem.sentence(), answer: faker.lorem.sentence() });
+
+  dialogId = dialog.body["_id"];
+});
+describe("Dialog consultation", () => {
+  it("Check the limit with a valid value", async () => {
+    const limit = 1;
+    const response = await request.get("/v1/dialog").set("limit", limit);
+
+    expect(response.statusCode).toBe(200);
+    expect(response.body).toHaveLength(1);
+  });
+  it("Check a threshold that receives an invalid value", async () => {
+    const response = await request
+      .get("/v1/dialog")
+      .set("limit", "invalidLimit");
+
+    expect(response.statusCode).toBe(200);
+    expect(response.body).toHaveLength(2);
+  });
+
+  it("Check the page with a valid value", async () => {
+    const page1 = await request
+      .get("/v1/dialog")
+      .set("limit", 1)
+      .set("page", 1);
+
+    const page2 = await request
+      .get("/v1/dialog")
+      .set("limit", 1)
+      .set("page", 2);
+
+    expect(page1.statusCode).toBe(200);
+    expect(page2.statusCode).toBe(200);
+    expect(page1.body[0]["_id"]).not.toBe(page2.body[0]["_id"]);
+  });
+
+  it("Check that the count was correct", async () => {
+    const response = await request.get("/v1/dialog");
+
+    expect(response.statusCode).toBe(200);
+    expect(response.header["x-total-count"]).toBe("2");
+  });
+  it("Check all fields of the dialog", async () => {
```

Comment:
In fact you are testing the queries of the dialogue and not the fields.

```suggestion
  it("Check all queries of the dialog", async () => {
```

Create a new 'it' to check a query without any queries expecting a status of 200 and all the correct fields in the body of the response.